x/ibc/03-connection/keeper/handshake.go

```diff
@@ -139,12 +147,15 @@ func (k Keeper) ConnOpenAck(
 	proofHeight uint64,
 	consensusHeight uint64,
 ) error {
-	// XXX: blocked by #5078
-	/*
-		if consensusHeight > uint64(ctx.BlockHeight()) {
-			return sdkerrors.Wrap(ibctypes.ErrInvalidHeight, "invalid consensus height")
-		}
-	*/
+	consHeight := int64(consensusHeight)
+
+	if consHeight > ctx.BlockHeight() {
```

Comment:
It can't be equal to the current height, either, since we're running that block, right?